load.php

```diff
@@ -291,13 +300,23 @@ function perflab_is_standalone_plugin_loaded( $module ) {
  * Gets the standalone plugin constants used for each module / plugin.
  *
  * @since 2.2.0
+ * @since n.e.x.t The `$type` parameter was added.
  *
- * @return array Map of module path to version constant used.
+ * @param string $type Optional. Either 'modules' or 'plugins'. Default 'modules'.
+ * @return array Map of module path / plugin slug and the version constant used.
```

Comment:
Not important, but might as well.

```suggestion
 * @return array<string, string> Map of module path / plugin slug and the version constant used.
```